spec/support/koala_test.rb

```diff
@@ -241,8 +241,8 @@ def self.activate_adapter!
           require 'typhoeus/adapters/faraday' if adapter.to_s == "typhoeus"
           Faraday.default_adapter = adapter.to_sym
         end
-      rescue ParserError
```

Comment:
ParserError doesn't seem to be a valid Ruby class anymore. This wasn't spotted before because tests need to be run with LIVE=1 which isn't part of CI